cypress/components/flat-table/flat-table.cy.tsx

```diff
@@ -2274,6 +2274,47 @@ context("Tests for Flat Table component", () => {
       flatTableBodyRowByPosition(5).should("be.focused");
     });
 
+    it("should set tabIndex 0 on the first row when after the loading state has finished", () => {
+      CypressMountWithProviders(<stories.KeyboardNavigationWithPagination />);
+      cy.wait(300);
+
+      cy.get("body").tab();
+
+      cy.focused().tab();
+      flatTableBodyRowByPosition(0).then(checkNewFocusStyling);
+      cy.focused().tab();
+      flatTableBodyRowByPosition(0).should("not.be.focused");
```

Comment:
**question** (non-blocking): I'm a bit surprised we're explicitly checking rows 0, 1 and 3 for not being focused, but not the other two rows? (This after tabbing off the focused row, so  Seems a bit arbitrary to me.